WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/tab/WooPosTabShouldBeVisible.kt

```diff
@@ -0,0 +1,40 @@
+package com.woocommerce.android.ui.woopos.tab
+
+import com.woocommerce.android.tools.SelectedSite
+import com.woocommerce.android.ui.woopos.WooPosIsScreenSizeAllowed
+import com.woocommerce.android.util.IsRemoteFeatureFlagEnabled
+import com.woocommerce.android.util.RemoteFeatureFlag.WOO_POS
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import org.wordpress.android.fluxc.store.WooCommerceStore
+import javax.inject.Inject
+
+class WooPosTabShouldBeVisible @Inject constructor(
+    private val selectedSite: SelectedSite,
+    private val isScreenSizeAllowed: WooPosIsScreenSizeAllowed,
+    private val wooCommerceStore: WooCommerceStore,
+    private val isRemoteFeatureFlagEnabled: IsRemoteFeatureFlagEnabled
+) {
+    suspend operator fun invoke(): Boolean = withContext(Dispatchers.IO) {
+        val selectedSite = selectedSite.getOrNull() ?: return@withContext false
+
+        if (!isRemoteFeatureFlagEnabled(WOO_POS)) return@withContext false
+
+        if (!isScreenSizeAllowed()) return@withContext false
+
+        val siteSettings = wooCommerceStore.getSiteSettings(selectedSite)
+            ?: wooCommerceStore.fetchSiteGeneralSettings(selectedSite).model
+
+        if (siteSettings == null) return@withContext false
+
+        return@withContext isCountrySupported(
+            countryCode = siteSettings.countryCode
+        )
+    }
+
+    private fun isCountrySupported(countryCode: String) = SUPPORTED_COUNTRIES.contains(countryCode.lowercase())
+
+    private companion object {
+        val SUPPORTED_COUNTRIES = listOf("us", "gb")
```

Comment:
```suggestion
        private val SUPPORTED_COUNTRIES = listOf("us", "gb")
```
It's not currently used outside this class.

Reply:
❓ In the existing `WooPosIsEnabled` use case, we're checking currencies in addition to country codes. I just want to confirm this is intentional to skip the currency check.

Reply:
> ❓ In the existing WooPosIsEnabled use case, we're checking currencies in addition to country codes. I just want to confirm this is intentional to skip the currency check.
Thanks for confirming this one! Yeah, we show the tab for those stores in eligible countries even if their currency is not the allowed one. We want to let them know once they visit the tab that by simply changing their currency, they can access the POS.